### DISCUSSION TITLE
评论后时间显示为8小时

### DISCUSSION THREAD

**Discussion by xlenco** [walinejs/waline#826] [Q&A] [answered]
![VeryCapture_20220221203229](https://user-images.githubusercontent.com/75837117/154955954-e0a433b5-5c09-466c-99b7-ed4070b3d985.gif)
评论后时间显示为8小时是怎么回事呀？

**Comment by lizheming** [comment]
提供下部署方式

**Reply by xlenco** [reply]
railway

**Reply by lizheming** [reply]
提供下存储方式

**Reply by xlenco** [reply]
github

**Reply by lizheming** [reply]
那可能需要看下存储后的时间有没有问题

**Comment by xlenco** [comment]
有解决方法吗？

**Comment by Mister-Hope** [comment, marked as answer]
这个很大可能的原因是时区不同，比如你的服务端使用UTC时间，但数据库和用户所在地区使用北京时间，导致出现了8个小时的时差。

这个只能你自己配置，因为对大多数服务来说，他们都统一使用UTC。

**Reply by elainecyx** [reply]
一般服务器使用的是UTC时间，浏览器应该根据所在的地区显示当地时区的时间，这样才合理，所以前端页面的显示应该加上或者减去时区

**Reply by lizheming** [reply]
https://github.com/walinejs/waline/pull/1875 第二个修复，需要等 client v3 版本

**Reply by Mister-Hope** [reply]
所以我是不是需要加时区？

**Reply by lizheming** [reply]
应该不用，之前 client 是 parse string 解析的时间，现在直接解析 timestamp 就行
